From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Price <greg@zulip.com>
Date: Thu, 16 Jun 2022 21:18:59 -0700
Subject: [tsflower] rnav: Delete files we don't use, that require
 r-n-masked-view

These files want to import `@react-native-masked-view/masked-view`,
which is a library we don't use and don't have in our dependencies.
(It isn't among the dependencies of this react-navigation library,
either.)  So naturally we have no types for it.

Conveniently, nothing imports these files.  So we can just delete all
these files entirely, and then Flow is happy.

This is marked `[tsflower]` just because it'd be a reasonable TsFlower
feature to have a way to configure it to ignore subsets of the files in
the input.  Then we'd ignore these in the first place instead of having
this patch to delete the corresponding output files.
---
 .../typescript/src/MaskedView.android.js.flow |  4 ----
 .../lib/typescript/src/MaskedView.ios.js.flow |  4 ----
 .../lib/typescript/src/MaskedView.js.flow     | 16 --------------
 .../typescript/src/MaskedViewNative.js.flow   | 21 -------------------
 4 files changed, 45 deletions(-)
 delete mode 100644 types/@react-navigation/elements/lib/typescript/src/MaskedView.android.js.flow
 delete mode 100644 types/@react-navigation/elements/lib/typescript/src/MaskedView.ios.js.flow
 delete mode 100644 types/@react-navigation/elements/lib/typescript/src/MaskedView.js.flow
 delete mode 100644 types/@react-navigation/elements/lib/typescript/src/MaskedViewNative.js.flow

diff --git types/@react-navigation/elements/lib/typescript/src/MaskedView.android.js.flow types/@react-navigation/elements/lib/typescript/src/MaskedView.android.js.flow
deleted file mode 100644
index 37ab18994..000000000
--- types/@react-navigation/elements/lib/typescript/src/MaskedView.android.js.flow
+++ /dev/null
@@ -1,4 +0,0 @@
-/* @flow
- * @generated by TsFlower
- */
-export { default } from './MaskedViewNative';
diff --git types/@react-navigation/elements/lib/typescript/src/MaskedView.ios.js.flow types/@react-navigation/elements/lib/typescript/src/MaskedView.ios.js.flow
deleted file mode 100644
index 37ab18994..000000000
--- types/@react-navigation/elements/lib/typescript/src/MaskedView.ios.js.flow
+++ /dev/null
@@ -1,4 +0,0 @@
-/* @flow
- * @generated by TsFlower
- */
-export { default } from './MaskedViewNative';
diff --git types/@react-navigation/elements/lib/typescript/src/MaskedView.js.flow types/@react-navigation/elements/lib/typescript/src/MaskedView.js.flow
deleted file mode 100644
index 45d000a19..000000000
--- types/@react-navigation/elements/lib/typescript/src/MaskedView.js.flow
+++ /dev/null
@@ -1,16 +0,0 @@
-/* @flow
- * @generated by TsFlower
- */
-import type { JSXElementConstructor as $tsflower_subst$React$JSXElementConstructor } from 'tsflower/subst/react';
-import * as React from 'react';
-
-type Props = {
-  maskElement: React$Element<React$ElementType>,
-  children: React$Element<React$ElementType>,
-  ...
-};
-
-declare export default function MaskedView(
-  Props,
-): React$Element<string | $tsflower_subst$React$JSXElementConstructor<any>>;
-export {};
diff --git types/@react-navigation/elements/lib/typescript/src/MaskedViewNative.js.flow types/@react-navigation/elements/lib/typescript/src/MaskedViewNative.js.flow
deleted file mode 100644
index 39da71d49..000000000
--- types/@react-navigation/elements/lib/typescript/src/MaskedViewNative.js.flow
+++ /dev/null
@@ -1,21 +0,0 @@
-/* @flow
- * @generated by TsFlower
- */
-import typeof * as $tsflower_import_typeof$_$_40_react_2d_native_2d_masked_2d_view_2f_masked_2d_view from '@react-native-masked-view/masked-view';
-
-import type {
-  ComponentProps as $tsflower_subst$React$ComponentProps,
-  JSX$Element as $tsflower_subst$React$JSX$Element,
-} from 'tsflower/subst/react';
-
-import * as React from 'react';
-type MaskedViewType = $ElementType<
-  $tsflower_import_typeof$_$_40_react_2d_native_2d_masked_2d_view_2f_masked_2d_view,
-  'default',
->;
-type Props = $tsflower_subst$React$ComponentProps<MaskedViewType> & {
-  children: React$Element<React$ElementType>,
-  ...
-};
-declare export default function MaskedView(Props): $tsflower_subst$React$JSX$Element;
-export {};
-- 
2.32.0

